AppCenterDistribute/AppCenterDistribute/MSACDistributeDelegate.h

```diff
@@ -28,6 +28,14 @@ NS_SWIFT_NAME(DistributeDelegate)
  */
 - (BOOL)distribute:(MSACDistribute *)distribute releaseAvailableWithDetails:(MSACReleaseDetails *)details;
 
+/**
+ * Callback method that will be called before the app is permanently closed for update.
+ * It is the right place to add any required clean ups.
+ *
+ * @param distribute The instance of MSACDistribute.
+ */
+- (void)distributeWillClose:(MSACDistribute *)distribute;
```

Comment:
I am curious about why "close"? I suppose something more radical in this case could be more adequate like "terminate". Closing for me may convey that even though the UI closes the app may still be running in background. Just my thoughts.

Reply:
Or maybe "Terminate" might be confusing the OS lifcycle callback. "Exit" same as the HA name: https://github.com/bitstadium/HockeySDK-iOS/blob/develop/Classes/BITUpdateManagerDelegate.h#L158 is an option.

Reply:
Makes sense. Let's name it `distributeWillExitApp`, the same as it's in the HA SDK?